components/image/image.css

```diff
@@ -0,0 +1,16 @@
+@import url(https://feather-com.netlify.app/styles.css);
+
+.image-responsive{
+    height: auto;
+    max-width: 100%;
+    border-radius: 0.5rem;
+    padding: 1rem;
+}
+
+.image-round{
+    height: auto;
```

Comment:
different height and width will make your image oval shaped and not circle. You might want to change that to get a perfect round shape

Reply:
Ya, would change it.